racetrack_commons/racetrack_commons/api/tracing.py

```diff
@@ -4,27 +4,29 @@
 from fastapi import Request
 from exceptiongroup import ExceptionGroup
 
-from racetrack_client.log.exception import get_exc_info_details
-from racetrack_client.log.logs import get_logger
-from racetrack_commons.api.debug import is_env_flag_enabled
-
-logger = get_logger(__name__)
```

Comment:
Do you want to keep using `get_logger(__name__)` here as well?

Reply:
Good point, I'll switch to `get_logger` factory